tools/Dockerfiles/fedora_sandbox

```diff
@@ -0,0 +1,41 @@
+FROM fedora:rawhide
+
+# Install generic dependencies to build jss
+RUN true \
+        && dnf update -y --refresh \
+        && dnf install -y dnf-plugins-core gcc make rpm-build cmake \
+                          glassfish-jaxb-api nss-tools apache-commons-codec \
+                          apache-commons-lang gcc-c++ jpackage-utils slf4j \
+                          zlib-devel perl slf4j-jdk14 junit ninja-build \
+                          gyp gtest mercurial \
+        && dnf build-dep nss nspr jss -y \
```

Comment:
for `jss`, when you build-dep on the koji version, the CI will fail in case spec file is changed. Probably do a wget on the latest spec file? :)

Reply:
We actually specify all of the dependencies in the previous `dnf` statement, since this is cloned from the `fedora_rawhide` container image (which doesn't build an RPM and only builds and runs the JSS test suite regularly).

Since we already maintain dependency lists outside of the spec file (for Debian and Ubuntu also), maintaining them here isn't much additional work.

And no `wget` is necessary, since the current tree's build is linked into the container; I could point to the path of the actual spec file. Perhaps I should just remove the `jss` here. 

Note that NSS and NSPR's deps (via `builddep`) are incorrect; some of the previous `dnf` statements bring in NSS / NSPR sandbox build deps that aren't in the spec file.